### PR TITLE
pyml release 20190626

### PR DESCRIPTION
**Pull request by thierry-martinez**
- Support for debug build of Python library
  (Suggested by Arlen Cox:
   https://github.com/thierry-martinez/pyml/issues/18)
- Bug fix in pyml_check_symbol_available
- `Py.compile` is a wrapper for the built-in function `compile`
  (Suggested by Dhruv Makwana:
   https://github.com/thierry-martinez/pyml/issues/25)
- Guarantees for structural and physical equalities on `Py.Object.t`
  are now documented. New predicates Py.is_none, Py.is_null, Py.Bool.is_true,
  Py.Bool.is_false, Py.Tuple.is_empty.
  (Suggested by Laurent Mazare:
   https://github.com/thierry-martinez/pyml/pull/31)
- Fix Py.Array.numpy to handle OCaml GC's moving the floatarray
  (Reported by Ilias Garnier:
   https://github.com/thierry-martinez/pyml/issues/30)

### CODE REVIEW

File: packages/pyml/pyml.20190626/opam [hunk 1]
```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+maintainer: "Thierry Martinez <martinez@nsup.org>"
+authors: "Thierry Martinez <martinez@nsup.org>"
+homepage: "http://github.com/thierry-martinez/pyml"
+bug-reports: "http://github.com/thierry-martinez/pyml/issues"
+license: "BSD"
+dev-repo: "git+https://github.com/thierry-martinez/pyml.git"
+build: [make "all" "pymltop" "pymlutop" {utop:installed} "PREFIX=%{prefix}%"]
+install: [make "install" "PREFIX=%{prefix}%"]
+remove: [make "uninstall" "PREFIX=%{prefix}%"]
```
Comment (mseri) [inline packages/pyml/pyml.20190626/opam]:
```suggestion
remove: [make "uninstall" "PREFIX=%{prefix}%"]
```

This is no longer needed in opam2

**Comment by mseri** [comment]
I have a minor comment, then it's ready to merge imho. I would have made the edit myself but it is not allowing me to commit the change.

File: packages/pyml/pyml.20190626/opam [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+synopsis: "OCaml bindings for Python"
+description: "OCaml bindings for Python 2 and Python 3"
+depends: [
+  "ocaml" {>= "3.12.1" & < "4.09.0"}
+  "ocamlfind" {build}
+  "stdcompat" {>= "9"}
+  "num"
+]
+depopts: ["utop"]
+version: "20190626"
```
Comment (mseri) [inline packages/pyml/pyml.20190626/opam]:
```suggestion
```

This is redundant, opam will infer the version from the package folder name and override it

File: packages/pyml/pyml.20190626/opam (hunk 1 above)
Comment (thierry-martinez) [reply packages/pyml/pyml.20190626/opam]: Thank you! Fixed.

File: packages/pyml/pyml.20190626/opam (hunk 2 above)
Comment (thierry-martinez) [reply packages/pyml/pyml.20190626/opam]: Thank you! Fixed.

**Comment by thierry-martinez** [comment]
@mseri Thank you for the review!

**Comment by mseri** [comment]
Nothing, thank you very much

**Pull request merged**